test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -2067,6 +2067,22 @@ public synchronized Set<String> nodesInclude(String index) {
         return Collections.emptySet();
     }
 
+    public synchronized Set<String> nodesIDsInclude(String index) {
```

Comment:
There is a `nodesInclude()` already. Would that be sufficient instead of a new function?